ring/sampler_gaussian.go

```diff
@@ -19,10 +19,49 @@ func NewGaussianSampler(prng utils.PRNG, context *Context) *GaussianSampler {
 	gaussianSampler.context = context
 	gaussianSampler.prng = prng
 	gaussianSampler.randomBufferN = make([]byte, context.N)
+	gaussianSampler.sample = gaussianSampler.SampleLvl
 	return gaussianSampler
 }
 
-// Sample samples a truncated gaussian polynomial with variance
+// NewGaussianSampler creates a new instance of GaussianSampler.
+// Accepts a PRNG and context and samples different kinds of gaussian polynomials
+func NewGaussianSamplerLvl(prng utils.PRNG, context *Context) *GaussianSampler {
```

Comment:
To follow the same pattern as the other samplers, the `NewGaussianSampler` constructors should accept the `sigma` and `bound` parameters and they should be removed from the interface. 

@Pro7ech Correct me if I'm wrong, I think we agreed that the loss in flexibility was acceptable wrt streamlining the interface.